cli/app.py

```diff
@@ -63,9 +51,12 @@ def create(cls, app_name, config, args=None):
         """
         settings_dir = config.get('defaults', 'settings')
 
-        if app_name in ["provisioner", ]:
+        if app_name not in ["provisioner", "installer"]:
```

Comment:
Also, instead of "app", please use "spec", to avoid a terminology mix.
This factory creates objects from spec files == spec objects.